apply_extra.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Extract the appimage
+chmod +x httpie.appimage
+unappimage httpie.appimage
+
+# Move all the data to /app/extra/httpie
+mv squashfs-root httpie
+
+# Install icons
+for size in 16x16 32x32 64x64 128x128 256x256 512x512 1024x1024; do
```

Comment:
1024x1024 is an invalid size.

there is absolutely no point doing this in the apply_extra.

Reply:
> 1024x1024 is an invalid size.

There is a 1024x1024 icon inside of the AppImage. Should it just be ignored?

> there is absolutely no point doing this in the apply_extra.

The icons come from the AppImage, which only gets extracted in apply_extra, so I can't install the icons before that. ([That's how Lunar Client does that as well](https://github.com/flathub/com.lunarclient.LunarClient/blob/72e97269feba8a1835d0845f90e00ba71dad7c14/apply_extra.sh#L19)). How else could I install the icons?

Reply:
Add one here (preferably svg) then install it like the desktop file.

Reply:
I am already doing that with the 256x256 icon:

https://github.com/flexagoon/flathub/blob/647ca282dc321bac6f10cc17cb848b7b9ccf3cd0/io.httpie.Httpie.yaml#L69

The apply_extra script installs the icons in other sizes.

Reply:
Do you need icons of every size? If not then install 512px or an svg.

Reply:
Upstream doesn't have an svg icon, so I'll use 512px. What's wrong with installing icons in apply_extra though?

Reply:
and it's totally unecesssary because they are not exported

Reply:
remove line 10 to 13.